lib/puppet/provider/grafana_organization/grafana.rb

```diff
@@ -0,0 +1,119 @@
+#    Copyright 2015 Mirantis, Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
```

Comment:
is that actually needed? The whole project uses the Apache-2 license. We try to keep the header short and simple, and don't add the license text to all files.